qa/rpc-tests/wallet_listreceived.py

```diff
@@ -404,7 +360,7 @@ def test_received_sapling(self, height):
         assert_equal(r[0]['amount'], Decimal('4'))
         assert_equal(r[0]['amountZat'], 400000000)
         assert_equal(r[0]['confirmations'], 3)
-        assert r[0]['outindex'] < 2
+        assert 0 <= r[0]['outindex'] < 2
```

Comment:
I don't know what the semantics of this are in python; is a boolean being coerced to a number? What's the operator precedence here? I'd prefer it if it were `0 <= r[0]['outindex'] and r[0]['outindex'] < 2`

Reply:
I didn't know about it either until I recently saw this recently: https://youtu.be/M3GAJ1AIIlA
I can change it if you prefer, but I kind of like it :slightly_smiling_face:

Reply:
Ah. Okay, I guess. I'm not really a fan of magical syntax like this, but I guess that comparison operators are themselves a syntax sugaring in Python so maybe it's fine.

Reply:
What probably would have been better is to just leave the assert as it was, since a bug making `outindex` negative would be very unlikely.